### ISSUE TITLE
Arcade View is consuming AttributeError

### ISSUE THREAD

**Issue by einarf** [pvcraven/arcade#676]
This was fixed in the pyglet event loop, but `arcade.View` needs the same fix.
https://github.com/pvcraven/arcade/blob/74a8012a001229cee677acbf2a285ef677c8b691/arcade/application.py#L560-L729

Looks like all the callback methods are affected.

**Comment by pvcraven** [comment]
With the textbox? Same issue exists in Window class. Also related, issue #673. How does this come up? Someone developing on TextBox?

I'm tempted to ditch the GUI stuff in 2.4. Instead build as a separate project. I like the idea of developing it independently until it is mature.

**Comment by einarf** [comment]
I lot of the game jam participants was at least using the button stuff. I did not think about this when debugging and assumed it was consuming `AttributeError` regardless of gui buttons used or not. Need to look again.

The exception consuming was fixed in pyglet 1.5.4 and I made sure they upgraded.

Would it not be better to remove gui in 3.0 or something? 2.4 might be a bit premature considering a lot of the game jam people are using 2.4a5 with gui buttons?
